index.js

```diff
@@ -3,6 +3,13 @@
 //  Hints: What are the basics needed for an express app?
 //         What is missing to configure pug as the view engine
 const scores = require('./scores')
+const express = require('express')
+const app = express()
+
+const router = express.Router()
```

Comment:
Just worth a mention that when we use the shortcuts for the router of `app.get`, etc., we do not need to specifically reference `router` at all.